tests/zfs-tests/include/properties.shlib

```diff
@@ -10,11 +10,10 @@
 #
 
 #
-# Copyright (c) 2012, 2016 by Delphix. All rights reserved.
+# Copyright (c) 2012, 2016, Delphix. All rights reserved.
 #
 
-typeset -a compress_prop_vals=('on' 'off' 'lzjb' 'gzip' 'gzip-1' 'gzip-2'
-    'gzip-3' 'gzip-4' 'gzip-5' 'gzip-6' 'gzip-7' 'gzip-8' 'gzip-9' 'zle' 'lz4')
+typeset -a compress_prop_vals=('off' 'lzjb' 'lz4' 'gzip' 'zle')
```

Comment:
Are the gzip levels exercised elsewhere, or is this an intentional reduction in coverage?

Reply:
The most basic functionality is tested under "history_002_pos.ksh", for all aliasses and levels.
The actual levels should all provide the same coverage.

However, having 9 gzip levels creates 9 times the chance of a gzip level (when randomly selecting) in comparison to any other algorithm.

Considering the future might/will bring other level-based compression, having the levels in here is not sustainable.